amd64/root/start.sh

```diff
@@ -29,7 +34,11 @@ if [ "$DECONZ_VNC_MODE" != 0 ]; then
   fi
   
   # Set VNC password
-  echo "$DECONZ_VNC_PASSWORD" | tigervncpasswd -f > /root/.vnc/passwd
+  if [ "$DECONZ_VNC_PASSWORD_FILE" != 0 ]; then
+      cat "$DECONZ_VNC_PASSWORD_FILE" | tigervncpasswd -f > /root/.vnc/passwd
+  else
+    echo "$DECONZ_VNC_PASSWORD" | tigervncpasswd -f > /root/.vnc/passwd
```

Comment:
Maybe the file could be loaded into `DECONZ_VNC_PASSWORD` and than there is only one call to `tigervncpasswd`.
Depends on personal taste.

Reply:
Will be resolved in the next commit